src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.scss

```diff
@@ -1,3 +1,21 @@
+@media (max-width: 768px) { //applies only to mobile screens
+  .custom-buttons {
+    width: 115px;
+    height: 30px;
+    margin-bottom: 5px;
+    //margin-right: 5px;
+  }
+}
+
+@media (max-width: 768px) { //applies only to mobile screens
+  .custom-buttons2 {
+    width: 95px;
+    height: 30px;
+    margin-bottom: 5px;
+    //margin-right: 5px;
+  }
+}
+
```

Comment:
Placing a few comments together:
- If the above comment on the height of the button makes sense, let's remove both the `height`s here as well
- Do delete the commented line if it's not needed!
- Let's rename `.custom-buttons` as `.custom-button-active` and `.custom-buttons2` as `.custom-button-archived`. I think that'll be a bit more descriptive (don't forget to make the corresponding changes in the `.html` file)